woocommerce/wc-admin
# Refactors out summary and graph

/\*_ @format _/

Refactors out summary and graph into its own component so that they can be used across different report screens.

### Screenshots

![screen shot 2018-10-12 at 2 36 43 pm](https://user-images.githubusercontent.com/6817400/46887666-45d0e100-ce2c-11e8-8019-74e716edc582.png)

### Detailed test instructions:

 - Open: /wp-admin/admin.php?page=wc-admin#/analytics/orders?period=month&compare=previous_period&chart=net_revenue
 - Observe graph and summary

I am not sure what should be tested here and also how to test.  If you point me in the right direction I am happy to do so.

@ryelle did I place these components where you thought they should go?

Once this is merged I will transition Revenue to use the components.
